web/src/components/base/RadioElement.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { ReactElement, ReactNode } from 'react'
+import styled from 'styled-components'
+
+export const RadioGroupContainer = styled.fieldset`
+  border: none;
+  box-sizing: border-box;
+  margin: 20px 0;
+  padding: 0;
+  width: 100%;
+`
+
+const RadioGroupCaption = styled.legend`
+  font-weight: bold;
+  padding-bottom: 10px;
+`
+
+const Radio = styled.input`
+  width: 1.3em;
+  height: 1.3em;
+  align-self: center;
+  accent-color: ${props => props.theme.colors.textSecondaryColor};
+  margin-right: 20px;
+  flex-shrink: 0;
+`
+
+const RadioLabel = styled.label`
+  display: flex;
+`
+
+const RadioElementContainer = styled.div`
+  width: 100%;
+  padding: 12px 0;
+`
+
+type Option<T> = {
+  groupId: string
+  id: T
+  label: string
+  checked: boolean
+  onChange: (newValue: T) => void
+  children?: ReactNode
+}
+
+type RadioGroupProps = {
+  caption: string
+  children?: ReactNode | ReactNode[]
+}
+
+export const RadioGroup = ({ caption, children }: RadioGroupProps): ReactElement => (
+  <RadioGroupContainer>
+    <RadioGroupCaption>{caption}</RadioGroupCaption>
+    {children}
+  </RadioGroupContainer>
+)
+
+const RadioElement = <T extends string>({
+  groupId,
+  id,
+  label,
+  checked,
+  children,
+  onChange,
+}: Option<T>): ReactElement => (
+  <RadioElementContainer>
+    <RadioLabel htmlFor={label}>
+      <Radio
```

Comment:
Radio should have role="radio"

Reply:
For an input type="radio" you should not need the [aria role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/radio_role).
```
The first rule of ARIA is if a native HTML element or attribute has the semantics and behavior you require, use it instead of re-purposing an element and adding ARIA. Instead use the native [HTML <input type="radio">](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio) (with an associated [<label>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label), which natively provides all the functionality required
```

Reply:
Also https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/radio_role . `input type="radio"` implicitly har role "radio"